commit 04c38a93ed72148a8bb0e29e416af5a486a46d60
Author: Fabio Erculiani <lxnay@sabayon.org>
Date:   Tue Aug 16 22:00:00 2011 +0200

    gen_compile: always use -j1 with kernel *_install targets, fixes compilation with make 3.82

diff --git a/gen_compile.sh b/gen_compile.sh
index 0758c3a..01b20f8 100755
--- a/gen_compile.sh
+++ b/gen_compile.sh
@@ -288,7 +288,7 @@ compile_modules() {
 	compile_generic modules kernel
 	export UNAME_MACHINE="${ARCH}"
 	[ "${INSTALL_MOD_PATH}" != '' ] && export INSTALL_MOD_PATH
-	compile_generic "modules_install" kernel
+	MAKEOPTS="${MAKEOPTS} -j1" compile_generic "modules_install" kernel
 	unset UNAME_MACHINE
 }
 
@@ -312,7 +312,7 @@ compile_kernel() {
 	if [ -n "${firmware_in_kernel_line}" -a "${firmware_in_kernel_line}" != CONFIG_FIRMWARE_IN_KERNEL=y ]
 	then
 		print_info 1 "        >> Installing firmware ('make firmware_install') due to CONFIG_FIRMWARE_IN_KERNEL != y..."
-		compile_generic "firmware_install" kernel
+		MAKEOPTS="${MAKEOPTS} -j1" compile_generic "firmware_install" kernel
 	else
 		print_info 1 "        >> Not installing firmware as it's included in the kernel already (CONFIG_FIRMWARE_IN_KERNEL=y)..."
 	fi
